app/routes/resources+/images.tsx

```diff
@@ -0,0 +1,44 @@
+import { getImgResponse } from 'openimg/node'
+import { type Route } from './+types/images.ts'
+import { getDomainUrl } from '#app/utils/misc.tsx'
+
+export async function loader({ request }: Route.LoaderArgs) {
+	const domain = getDomainUrl(request)
+	const headers = new Headers()
+	headers.set('Cache-Control', 'public, max-age=31536000, immutable')
+	return getImgResponse(request, {
+		headers,
+		allowlistedOrigins: [domain, process.env.AWS_ENDPOINT_URL_S3],
```

Comment:
We don't actually fetch an image from Tigris directly just yet but I already added the `process.env.AWS_ENDPOINT_URL_S3` in the allowlist.